recipes/libxml2/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+if [ $(uname) == Darwin ]; then
+  export OPTS="--disable-dependency-tracking"
```

Comment:
If you type in

```bash
sbamac:~ balke$ brew install libxml2 --HEAD
==> Cloning https://git.gnome.org/browse/libxml2
Cloning into '/Library/Caches/Homebrew/libxml2--git'...
remote: Counting objects: 41849, done.
remote: Compressing objects: 100% (10640/10640), done.
remote: Total 41849 (delta 33453), reused 37699 (delta 30430)
Receiving objects: 100% (41849/41849), 17.48 MiB | 3.54 MiB/s, done.
Resolving deltas: 100% (33453/33453), done.
Checking connectivity... done.
==> Checking out branch master
==> ./autogen.sh
```

Otherwise it will take a bottle. You can also force brew to build from source.